### DISCUSSION TITLE
Storage location parent change and tracking

### DISCUSSION THREAD

**Discussion by Petrox** [inventree/InvenTree#5436] [Ideas]
- Storage locations themselves could move over time.
 - A vehicle could be a storage location sometimes
 - Their location could have a history

Scenario:
 - company has a toolbox with 53 parts inside
 - company might even have 16 sets of this toolbox
 - they create storage location for every toolbox
 - user picks up toolbox12. 
 - user does not want to scan every single item inside, just want to take it somewhere
 - user would like to change the parent of the storage location from "warehouse1" to "companycar5"
 - at the end of the day it can be put back to warehouse1
And this should leave a trace which should be easily accessible by anyone looking for it: "it's not clean, who used this yesterday?"

**Comment by SchrodingersGat** [comment]
I believe this is probably a duplicate of https://github.com/inventree/InvenTree/issues/2058

It is a good idea, would probably require a refactor to use the "history" functionality currently supported for stock items

**Reply by Petrox** [reply]
True, #2058 seems like a proper issue.
